packages/docs/src/components/DocSiteNavigation.tsx

```diff
@@ -88,8 +96,8 @@ const DocSiteNavigation = ({ location }: DocSiteNavProps) => {
   const data = useStaticQuery(graphql`
     query SiteNavQuery {
       allFile(
-        sort: { fields: [relativeDirectory, name] }
         filter: { ext: { eq: ".mdx" }, relativeDirectory: { ne: "not-in-sidebar" } }
+        sort: { fields: [relativeDirectory, name] }
```

Comment:
Not super important but we could remove `name` `relativeDirectory` and both `id`'s from this graphql query.